doc/release-notes/0.27/0.27.md

```diff
@@ -0,0 +1,96 @@
+<!--
+* Copyright (c) 2021, 2021 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.27.0 release notes
+
+These release notes support the [Eclipse OpenJ9 0.27.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.27.0/plan).
+
+## Supported environments
+
+OpenJ9 release 0.27.0 supports OpenJDK 8, 11, and 16.
+
+All releases are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests at AdoptOpenJDK.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.27.0. Further information about these changes can be found in the [user documentation](https://eclipse-openj9.github.io/openj9-docs/version0.27/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top">
+<a href="https://github.com/eclipse-openj9/openj9/issues/11999">#11999</a></td>
+<td valign="top">New <strong>-XX:[+|-]AdaptiveGCThreading</strong> option added</td>
+<td valign="top">All versions</td>
+<td valign="top">To optimize performance, you can enable adaptive threading to automatically tune the number of active parallel garbage collection (GC) threads.</td>
```

Comment:
It is actually enabled by default.

If we want to be more specific (rather than 'optimize perf'), we can say things like 'reduce pause time by Stop-The-World GC operations' (or some shorter variant of it).

Reply:
I've updated it to "Adaptive threading is enabled by default, which automatically tunes the number of active parallel garbage collection (GC) threads."

The docs https://eclipse-openj9.github.io/openj9-docs/version0.27/ will need updates as well. I'll add the additional perf information there, and get a review from you.